graphql_api/types/comparison/comparison.py

```diff
@@ -252,10 +252,6 @@ def resolve_has_different_number_of_head_and_base_reports(
     if "comparison" not in info.context:
         return False
     comparison: Comparison = info.context["comparison"]
-    try:
```

Comment:
Didnt specify but this is implicitly moved in the try/except inside the resolver logic